### PR TITLE
docker swarm

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: docs/manual-EN/3.build-develop-and-administration/3.configurations/0.system-requirement.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ None of the above processes need to monopolize a single machine. For example, a
 * E: storaged, graphd
 
 > Do not deploy the same cluster across two IDCs.
-> Each metad process automatically creates and maintains a copy of the metadata, so usually only 3 metead processes are needed. Meanwhile, the number of storaged processes does not affect the copy count of a graph space.
+> Each metad process automatically creates and maintains a copy of the metadata, so usually only 3 meted processes are needed. Meanwhile, the number of storaged processes does not affect the copy count of a graph space.
```
Comment (randomJoe211) [inline docs/manual-EN/3.build-develop-and-administration/3.configurations/0.system-requirement.md:20]: Maybe check again~

**Review by randomJoe211 — CHANGES_REQUESTED** [review]

File: docs/manual-EN/3.build-develop-and-administration/3.configurations/0.system-requirement.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Take AWS EC2 c5d.xlarge as an example:
 
 ## About HDD and Gigabit Networks
 
-**Nebula Graph** is designed for NVMe SSD and 10 Gigabit Network. There is no special adaptation for HDD and gigabit networks. The following are some parameters to be tuned:
+Nebula Graph is designed for NVMe SSD and 10 Gigabit Network. There is no special adaptation for HDD and gigabit networks. The following are some parameters to be tuned:
```
Comment (oldLady344) [inline docs/manual-EN/3.build-develop-and-administration/3.configurations/0.system-requirement.md:63]: is designed for NVMe SSD and 10 Gigabit Network. ==> is intended for NVMe SSD and 10 Gigabit Network.

I think "intended" may be more accurate?

**Review by whitewum — DISMISSED** [review]

File: docs/manual-EN/3.build-develop-and-administration/3.configurations/0.system-requirement.md (hunk 2 above)
Comment (whitewum) [reply docs/manual-EN/3.build-develop-and-administration/3.configurations/0.system-requirement.md:63]: I think both are ok

File: docs/manual-EN/3.build-develop-and-administration/2.install/deploy-nebula-with-swarm.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
+# Deploy Nebula Graph cluster with Docker Swarm
+
+This document gives the introduction on deploying a Nebula Graph cluster with Docker Swarm.
+
+## Prerequisites
+
+Before deploying cluster, make sure that you have installed Nebula Graph, Docker, Docker Compose. To customize load balancing and high availability, you need to install HAProxy, and Keepalived.
```
Comment (oldLady344) [inline docs/manual-EN/3.build-develop-and-administration/2.install/deploy-nebula-with-swarm.md:7]: Docker, Docker Compose.--> Docker, and Docker Compose.

**Review by whitewum — APPROVED** [review]

File: docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/create-edge-syntax.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ CREATE EDGE [IF NOT EXISTS] <edge_name>
     | DEFAULT <default_value>
 ```
 
-**Nebula Graph**'s schema is composed of tags and edges, either of which may have properties. `CREATE EDGE` statement defines an edge type with the given name.
+The schema for Nebula Graph is composed of tags and edges, either of which can have properties. The `CREATE EDGE` statement defines an edge type with the given name.
```
Comment (randomJoe211) [inline docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/create-edge-syntax.md:20]: "The schema for Nebula Graph is composed of tags and edges, and both of them can have properties"?

File: docs/manual-EN/3.build-develop-and-administration/2.install/deploy-nebula-with-swarm.md [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+
+| IP            | Memory (GB) | CPU (CORES) | Role    |
+| ------------- | ----------- | ----------- | ------- |
+| 192.168.1.166 | 16          | 4           | manager |
+| 192.168.1.167 | 16          | 4           | worker  |
+| 192.168.1.168 | 16          | 4           | worker  |
+
+## Create Nebula Graph cluster
+
+### Initialize the Docker Swarm cluster
```
Comment (randomJoe211) [inline docs/manual-EN/3.build-develop-and-administration/2.install/deploy-nebula-with-swarm.md:19]: Please consider transforming H3 titles in the Create Nebula Graph cluster section into an ordered list. That should provide a clearer view and the readers will know the execution orders at a glance.
The steps below the H3 titles can be substeps.

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**